wiring/inc/spark_wiring_usartserial.h

```diff
@@ -51,12 +61,13 @@ class USARTSerial : public Stream
   virtual int peek(void);
   virtual int read(void);
   virtual void flush(void);
+  virtual size_t write(uint16_t);
```

Comment:
The `write` 16-bit overload doesn't need to be virtual since it's not part of the `Stream` interface. (If we ever made a WideCharStream then it would be virtual, but let's cross that bridge when we get to it! ;-))